File: Singleton/Singleton/ViewController.m
//
//  ViewController.m
//  Singleton
//
//  Created by LMinh on 8/8/18.
//  Copyright © 2018 LMinh. All rights reserved.
//

#import "ViewController.h"
#import "NetworkManager.h"

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
    
    // Access NetworkManager
    NSLog(@"%@", [NetworkManager sharedInstance].baseURL);
}

@end

File: Singleton/Singleton/ViewController.swift
//
//  ViewController.swift
//  Singleton
//
//  Created by LMinh on 8/8/18.
//  Copyright © 2018 LMinh. All rights reserved.
//

import UIKit

class ViewController: UIViewController
{
    override func viewDidLoad()
    {
        super.viewDidLoad()
        
        // Access NetworkManager
        if let baseURL = NetworkManager.sharedInstance.baseURL {
            print(baseURL)
        }
    }
}

